src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1843,7 +1845,7 @@ Entered\ database\ has\ obsolete\ structure\ and\ is\ no\ longer\ supported.=Ent
 However,\ a\ new\ database\ was\ created\ alongside\ the\ pre-3.6\ one.=However, a new database was created alongside the pre-3.6 one.
 Opens\ a\ link\ where\ the\ current\ development\ version\ can\ be\ downloaded=Opens a link where the current development version can be downloaded
 See\ what\ has\ been\ changed\ in\ the\ JabRef\ versions=See what has been changed in the JabRef versions
-Referenced\ citation\ key\ '%0'\ does\ not\ exist=Referenced citation key '%0' does not exist
```

Comment:
Please do not remove features.

Think more of how this parameter can still be kept when using `IntegrityIssue`. Maybe, the localization input sring needs to be stored and the tests need to be adapted.